### PR TITLE
Chore/cleanup tools

### PR DESCRIPTION
**Pull request by muXxer**

### CODE REVIEW

File: pkg/toolset/database_hash.go [hunk 1]
```diff
@@ -122,25 +128,67 @@ func calculateDatabaseLedgerHash(dbStorage *storage.Storage) error {
...
+			Treasury:               treasury,
+			LedgerIndex:            ledgerIndex,
+			SnapshotIndex:          snapshotInfo.SnapshotIndex,
+			UTXOsCount:             len(outputs),
+			SEPsCount:              len(solidEntryPoints),
+			LedgerStateHash:        hex.EncodeToString(snapshotHashSumWithoutSEPs),
+			LedgerStateHashWithSEP: hex.EncodeToString(snapshotHashSumWithSEPs),
+		}
+
+		output, err := json.MarshalIndent(result, "", "  ")
```
Comment (alexsporn) [inline pkg/toolset/database_hash.go]: should we refactor this into a `func printJSON(obj interface{})` and use it in every tool?

File: pkg/toolset/database_migration.go [hunk 2]
```diff
@@ -8,42 +8,65 @@ import (
...
+	flag "github.com/spf13/pflag"
 
 	"github.com/gohornet/hornet/pkg/database"
 	"github.com/gohornet/hornet/pkg/utils"
 	"github.com/iotaledger/hive.go/configuration"
 	"github.com/iotaledger/hive.go/kvstore"
 )
 
+const (
+	FlagToolDatabaseMigrationDatabasePathSource   = "databasePathSource"
```
Comment (alexsporn) [inline pkg/toolset/database_migration.go]: I like it more like it was before `sourceDatabasePath`

File: pkg/toolset/jwt.go [hunk 3]
```diff
@@ -19,33 +19,31 @@ import (
...
 		return err
 	}
 
-	// Check if all parameters were parsed
-	if fs.NArg() != 0 {
-		fs.Usage()
-		os.Exit(2)
-	}
-
 	salt := nodeConfig.String(restapi.CfgRestAPIJWTAuthSalt)
```
Comment (alexsporn) [inline pkg/toolset/jwt.go]: do we know if this work with `--config`? Else better remove the config.json dependance altogether and let the users pass the required values as flags

File: pkg/toolset/snap_merge.go [hunk 4]
```diff
@@ -2,60 +2,153 @@ package toolset
...
 
+	flag "github.com/spf13/pflag"
+
+	"github.com/gohornet/hornet/pkg/model/milestone"
 	"github.com/gohornet/hornet/pkg/snapshot"
 	"github.com/iotaledger/hive.go/configuration"
 )
 
+const (
+	FlagToolSnapshotMergeSnapshotPathTarget = "snapshotPathTarget"
```
Comment (alexsporn) [inline pkg/toolset/snap_merge.go]: `targetSnapshotPath`

**Review by alexsporn — APPROVED** [review]

File: pkg/toolset/jwt.go (hunk 3 above)
Comment (muXxer) [reply pkg/toolset/jwt.go]: I removed it!

**Pull request merged**